### DISCUSSION TITLE
Is ok to make the video object stable?

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#2382] [Archive] [answered]
Video object exist since April 2019, and has been used lot of time since it was released as experimental.
Unless we have a good reason to keep it experimental, it has been tested a bunch of time by users, and deserve to join the stable objets. And make a free place for Bitmap Text object ;)

**Comment by 4ian** [comment]
Totally! Happy to make it stable 👍 Feel free to do the PR for it.

**Comment by Bouh** [comment, marked as answer]
Experimental tag removed https://github.com/4ian/GDevelop/commit/92c5cfd9c72dbf810fcbb2bda5130f559d73a377
